src/reachy2_sdk/parts/head.py

```diff
@@ -10,34 +10,25 @@
 import numpy as np
 from google.protobuf.wrappers_pb2 import FloatValue
 from pyquaternion import Quaternion as pyQuat
-from reachy2_sdk_api.goto_pb2 import (
-    CartesianGoal,
-    GoToAck,
-    GoToId,
-    GoToRequest,
-    JointsGoal,
-)
+from reachy2_sdk_api.goto_pb2 import CartesianGoal, GoToId, GoToRequest, JointsGoal
 from reachy2_sdk_api.goto_pb2_grpc import GoToServiceStub
 from reachy2_sdk_api.head_pb2 import Head as Head_proto
 from reachy2_sdk_api.head_pb2 import (
     HeadState,
     NeckCartesianGoal,
     NeckJointGoal,
     NeckOrientation,
-    SpeedLimitRequest,
-    TorqueLimitRequest,
 )
 from reachy2_sdk_api.head_pb2_grpc import HeadServiceStub
 from reachy2_sdk_api.kinematics_pb2 import ExtEulerAngles, Point, Quaternion, Rotation3d
-from reachy2_sdk_api.part_pb2 import PartId
 
 from ..orbita.orbita3d import Orbita3d
-from ..orbita.orbita_joint import OrbitaJoint
-from ..utils.custom_dict import CustomDict
 from ..utils.utils import get_grpc_interpolation_mode
+from .goto_based_part import GoToBasedPart
+from .joints_based_part import JointsBasedPart
 
 
-class Head:
+class Head(GoToBasedPart, JointsBasedPart):
```

Comment:
I'm not sure this is ideal from a design point of view. You have multiple inheritance here, but both GoToBasedPart and JointsBasedPart inherits from Part (see Diamond Problem https://en.wikipedia.org/wiki/Multiple_inheritance).
If I understand correctly they are both only used together by Head and Arm. So one option is to simply merge them into one class. 
Or you make one of them inherit from the other like Part -> JointsBasedPart -> GoToBasedPart -> Arm. If you feel that JointsBasedPart and GoToBasedPart needs to be used independently in a near future.
Or you make GoToBasedPart an interface (i.e. inheriting from ABC in python) so you can keep the "double inheritance" which can be seen as a simple inheritance + interface implementation.